Topics/Tech_Stacks/Postman_Backend_Testing.md

```diff
@@ -0,0 +1,64 @@
+
+# Postman Backend Testing
+
+In this Postman application overview, we will go over how to 
+install the application and the details of how to tests your API endpoints.
+
+## Download and installation
+the following link will take you to the postman website where you can
+download the application if you do not have it already installed:
+[Download](https://www.postman.com/downloads/)
+
+
+
+After the download the following video will help you install the application:
+
+[Windows](https://www.youtube.com/watch?v=pBo_oClYjjM&ab_channel=GeekyScript)  
+[Mac](https://www.youtube.com/watch?v=sriayOzVaMM&ab_channel=Postman)  
+[Linux](https://www.youtube.com/watch?v=9ZZdl4CQbCg&ab_channel=OSTechHelp)
+
+
+## Testing with Postman
+### Creating a workspace
+Before starting to test your API, it is best to create a workspace for the 

```

Comment:
The process of creating a workspace requires logging in and a bit of installation time, so it may not be necessary for someone who only needs to test a few APIs. Providing readers with the option to skip this step if they wish would be the best approach.